pygments/formatters/latex.py

```diff
@@ -97,6 +97,10 @@ def escape_tex(text, commandprefix):
 
 STYLE_TEMPLATE = r'''
 \makeatletter
+%% workaround a minted <= v2.5 issue, which loads this file with - having
+%% catcode letter; also with _ having catcode 11 but this does not affect us
+\edef\%(cp)s@restore@catcode@of@minus@sign{\catcode`\noexpand\-=\the\catcode`\-\relax}
+\catcode`\-=12
```

Comment:
defining the restoring macro is for maximal compatibility but anyhow regarding minted problem with `-` of catcode 11 it seems it happens within a group so it is unneeded to restore at end of this style file the catcode `-` to its (erroneous) minted setting anyhow